tools/xcodeproj_shims/build-wrapper.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -euxo pipefail
 
-$BAZEL_PATH build $1 2>&1 | $BAZEL_OUTPUT_PROCESSOR
+$BAZEL_PATH build --build_event_text_file=$bazel_build_event_text_filename --build_event_publish_all_actions $1 2>&1 | $BAZEL_OUTPUT_PROCESSOR
```

Comment:
I considered having the output-processor read from the bazel build event text file instead. But then I realized that we want to show error messages in the output as they appear, rather than waiting until the build is finished.